experiments/ClimaEarth/components/land/climaland_bucket.jl

```diff
@@ -138,6 +139,48 @@ function bucket_init(
     Y.bucket.Ws .= 0.0
     Y.bucket.σS .= 0.0
 
+    # Overwrite initial conditions with interpolated values from a netcdf file using
+    # the `SpaceVaryingInputs` tool. We expect the file to contain the following variables:
```

Comment:
It would be good to add an assertion checking that the file does contain all these variables, and provide an informative message if not